### ISSUE TITLE
ActiveStorage: DirectUploads - 'undefined' instead item.signed_id in params

### ISSUE THREAD

**Issue by mesmerze** [rails/rails#32365]
### Steps to reproduce

I want to add the incredible ActiveStorage feature to my project, but I've faced a problem with direct uploads.
I've used [rails guides](http://edgeguides.rubyonrails.org/active_storage_overview.html) for setup.

### Expected behavior
After the direct upload is done activestorage.js should put signed_id to the hidden input value and submit a form.
### Actual behavior

File saved, everything is fine, but the value of the hidden input:
```<input type="hidden" name="meeting[documents][]" value="undefined">```
and form get submitted.

I've found what ```i.signed_id``` returns `undefined` here:
```
this.directUpload.create(function(n, i) {
        n ? (r.parentNode.removeChild(r), e.dispatchError(n)) : r.value = i.signed_id,e.dispatch("end"),t(n)}
```
And changed this to ```i.blob.signed_id```, what (I'm not sure) fixes the problem.
Anyone else faced this problem? Or I broke something?

### System configuration
**Rails version**: 5-2-stable

**Ruby version**: ruby 2.4.1

**Comment by javan** [comment]
What does your response from `/rails/active_storage/direct_uploads` look like in the Network inspector? For example, here's what I see:

<img width="558" alt="screen shot 2018-04-09 at 6 30 37 am" src="https://user-images.githubusercontent.com/5355/38493640-de8b046a-3bc0-11e8-9eb0-a788993717eb.png">

**Comment by mesmerze** [comment]
HI! Here's what I see:
<img width="444" alt="screen shot 2018-04-09 at 13 46 29" src="https://user-images.githubusercontent.com/16397590/38493917-966b8592-3bfc-11e8-95f0-ec082e660d3f.png">

**Comment by javan** [comment]
Do you have `ActiveRecord::Base.include_root_in_json = true` set in your application?

**Comment by mesmerze** [comment]
> Do you have ActiveRecord::Base.include_root_in_json = true set in your application?
Yes

**Comment by georgeclaghorn** [comment]
No objection.

**Comment by memoht** [comment]
Leaving a comment here just in case. I ran into this error 
```
undefined method `signed_id' for #<ActiveStorage::Attached::One:0x0000000008e27ec8>
``` 
for a model with an attachment on the `Show` view. The User had edited the item in question to replace the attachment. In my app the above JSON setting is set to the default. Not sure what can lead to this scenario. The User didn't report any issues, so I only realized when another User tripped across the Show view.
